Tests/eppo/EppoValueTests.swift

```diff
@@ -68,6 +68,23 @@ class EppoValueTests: XCTestCase {
         XCTAssertFalse(arrayValue1 == arrayValueDifferent, "Arrays with different elements or counts should not be equal")
     }
 
+    func testToString() {
```

Comment:
same tests as in: https://github.com/Eppo-exp/python-sdk/pull/51

missing anything?

Reply:
👍 , we can always add more tests if we find more edge cases